### ISSUE TITLE
per-plugin and per-output READMEs

### ISSUE THREAD

**Issue by sparrc** [influxdata/telegraf#177]
We should have a README for each plugin and output, and new ones going forward should have it as well.

Could document each metric the plugin collects, at least in a general sense of the types and units of the measurements being collected

**Comment by sparrc** [comment]
@beckettsean ^^

**Comment by sparrc** [comment]
@beckettsean bump^

**Comment by beckettsean** [comment]
I think the README should follow a template as much as possible. A complete template should name the plugin, give version compatibility, describe the measurements, tags, and fields submitted (with units for the fields), and the write frequency. Here's a strawman proposal for that template:

For example:

```
# Telegraf Plugin: InfluxDB
# Versions Supported
0.9.0
0.9.1
0.9.2
0.9.3
0.9.4 (beta)
# Measurements
databases
- tag: name=<database_name>
- tag: node=<hostname>
- field: size=<size of db folder in KB>
- field: num_shards=<number of shards in the db on this node>
Submit frequency is set in config file

users
- tag: admin=<true | false>
- tag: user=<username>
- field: auth_succeeded=[ true | false ]
- field: auth_source=<IP or hostname source of auth attempt>
One point submitted for each login attempt

continuous_queries
- tag: name=<name of CQ>
- tag: database=<name of DB on which CQ created>
- field: run=true
- field: query=<executed query string>
One point submitted each time the CQ executes
```

**Comment by sparrc** [comment]
I've got my first draft up of a plugin README: https://github.com/influxdb/telegraf/blob/master/plugins/system/CPU_README.md

@beckettsean I made a few changes from your template so let me know what you think. I haven't included the InfluxDB versions because that shouldn't matter on a per-plugin basis. I also condensed the "measurements" list into groups, so that all measurements that have the same tags and units can be grouped together and take up less space.

This will of course change a bit when we implement #152, as all of these measurements will then become fields on the plugin measurement (some plugins may also have multiple measurements)
